From 94c27b7454b479c335fd83685b1e973ab6fdae2d Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 29 Nov 2023 23:50:00 +0100
Subject: [PATCH 69/N] fixup! Respect `db_home` setting even for the SYSTEM
 account

In preparation for integrating the patches that made it upstream via
https://inbox.sourceware.org/cygwin-patches/cover.1684753872.git.johannes.schindelin@gmx.de/,
let's revert the original patches from Git for Windows' branch thicket.

This reverts commit 22854f66a429d9bd583621f5b6e365fe9b88cd9d.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/uinfo.cc | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/winsup/cygwin/uinfo.cc b/winsup/cygwin/uinfo.cc
index b910878..1dbd826 100644
--- a/winsup/cygwin/uinfo.cc
+++ b/winsup/cygwin/uinfo.cc
@@ -2194,11 +2194,7 @@ pwdgrp::fetch_account_from_windows (fetch_user_arg_t &arg, cyg_ldap *pldap)
 	 it to a well-known group here. */
       if (acc_type == SidTypeUser
 	  && (sid_sub_auth_count (sid) <= 3 || sid_id_auth (sid) == 11))
-	{
-	  acc_type = SidTypeWellKnownGroup;
-	  home = cygheap->pg.get_home (pldap, sid, dom, domain, name,
-				       fully_qualified_name);
-	}
+	acc_type = SidTypeWellKnownGroup;
       switch ((int) acc_type)
 	{
 	case SidTypeUser:
